src/models/Card.ts

```diff
@@ -1,8 +1,18 @@
 import Label from './Label'
+import User from './User'
+import TasksList from './TasksList'
+import Comment from './Comment'
+import Attachment from './Attachment'
 
 export default interface Card {
-    title: string
-    labels: Label[]
-    percent: number
-    assignees: string[]
+    name: string,
```

Comment:
This is very different from the ICard type. Why ?

Reply:
Because of the new common routes made in the file Trello-Routes.